FlowersStore/Scenes/FlowersList/FlowersListInteractor.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FlowersListInteractor.swift
+//  FlowersStore
+//
+//  Created by Kauê Sales on 30/11/21.
+//
+
+import Foundation
+
+protocol FlowersListInteractorInput: AnyObject {
+    
+}
+
+protocol FlowersListInteractorOutput: AnyObject {
+    
+}
+
+class FlowersListInteractor {
+    private weak var output: FlowersListInteractorOutput?
```

Comment:
<!--
  0 failure: 
  1 warning:  Property 'output'...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListInteractor.swift;
  Line: 19;
-->

- :warning: Property 'output' is assigned, but never used